### DISCUSSION TITLE
以thorem环境为基础做\newthorem会在末尾自带一个证毕的小方格

### DISCUSSION THREAD

**Discussion by TBBnozomi** [nju-lug/NJUThesis#146] [Q&A] [answered]
我这里是做了个 `\newtheorem{conjecture}[theorem]{猜想}`，随后我使用这个conjecture环境会在输出的时候在环境的末尾加上一个小方格. 把theorem改成lemma, axiom之类的都不能解决这个问题, 这个选项似乎只是继承标号

**Comment by atxy-blip** [comment, marked as answer]
`\newtheorem` 前加一行 `\theoremsymbol{}` 清空证毕符号。

**Reply by TBBnozomi** [reply]
原来放前面就能生效

**Reply by atxy-blip** [reply]
v0.18 需要你把定制定理环境的代码放在 `\begin{document}` 后。
